xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -714,6 +714,12 @@ bool CGUIWindowVideoBase::OnItemInfo(int iItem)
     else
       strDir = URIUtils::GetDirectory(item->GetPath());
 
+    if (URIUtils::IsUPnP(strDir) && item->IsVideo())
+    {
+      CGUIDialogVideoInfo::ShowFor(*item);
```

Comment:
This should not be called from here. This is for external calls, and will in turn call back into CGUIWindowVideoBase::OnItemInfo